ftw/referencewidget/converter.py

```diff
@@ -145,6 +147,8 @@ def toWidgetValue(self, value):
         return new_value
 
     def toFieldValue(self, value):
+        if not value:
+            return []
```

Comment:
Let's also `return value` here: the value might be a persistent list and we don't want to destroy the persistence here 😉

Reply:
Done